lib/version_utils.pm

```diff
@@ -113,6 +114,13 @@ sub is_jeos {
     return get_var('FLAVOR', '') =~ /^JeOS/;
 }
 
+=head2 is_community_jeos
+
+Returns true if called on community jeos images
+=cut
+sub is_community_jeos {
```

Comment:
Only called once and can be simplified to `check_var('FLAVOR', 'JeOS-for-AArch64')`, so IMO not needed as a method

Reply:
True, I was not sure whether to go for a method or not as I cannot come up with another possible usage of it elsewhere in our code base. Could you recall any potential use case, please? 
Besides of this, I thought that it might be helpful for JeOS agnostic people to distinguish between our and community JeOS images.

Reply:
ah, ofc, I have overlook that one. Thanks!